doc/dependencies.csv

```diff
@@ -2,29 +2,22 @@ Build option/target,Tool name,Minimum version,Notes,Update criteria
 --,CMake,3.19,,"* Is available as a module on HPC systems
 * When arbitrary changes in more recently released versions make supported"
 --,C++ compiler,,C++17 support. See below,
---,GCC,9.0.0,,"* it is supported by the minimum version of CUDA.
+--,GCC,12.0.0,,"* it is supported by the minimum version of CUDA.
 * it is available either by default or can be installed using standard package manager on the supported Linux versions."
---,Clang,10.0,Needs GCC 9 or later for standard library.,
---,Apple Clang,11,Apple LLVM version 11.0.0 (clang-1100.0.33.8),
+--,Clang,12.0,,
+--,Apple Clang,13,Apple LLVM version 13.0.0 (clang-1300.x.y.z),
 ARB_GPU,Hip Clang,ROCm 3.9,HIP support is currently experimental.,
-ARB_GPU,CUDA,11.0,,"* It is available on all of the target HPC systems (Piz Daint, Juwels Booster)"
+ARB_GPU,CUDA,12.0,,"* It is available on all of the target HPC systems (Piz Daint, Juwels Booster)"
 ARB_WITH_MPI,MPI,,Many MPI implementations are supported.,
 "ARB_WITH_MPI, ARB_WITH_PYTHON ",mpi4py,3.1.0,Minimum version depends on compatiblity with your Python and MPI versions. 3.1.0 is the first release to support Python > 3.8.,
 html,     .. literalinclude:: ../requirements.txt,,See ``doc/requirements.txt``,
 unit,googletest,1.12.1,submodule ``ext/gtest.*``,
 bench,Google-benchmark,,submodule ``ext/google-benchmark``,
---,json,3.11.2,submodule ``ext/json``,
---,pugi,1.13,submodule ``ext/pugixml``,
+--,json,3.11.3,submodule ``ext/json``,
+--,pugi,1.14,submodule ``ext/pugixml``,
```

Comment:
what are the cretiria for these dependencies? Where do I find the most recent versions?

Reply:
These bundled libraries should be installable via spack, thus we must not be more recent then
the latest version there. If this is not the case, the CI workflow for spack will fail, so we got an
automated check there. 

Other than that, we try to track the latest stable releases.